docs/datacard.py

```diff
@@ -607,3 +641,29 @@ def limit_presented_samples(
             + samples[-limit // 2:]
         )
     return samples
+
+
+def set_plot_margins(
```

Comment:
This function can be re-used then for other plots, compare https://github.com/audeering/datasets/pull/5